lib/awful/mouse/init.lua

```diff
@@ -155,7 +155,10 @@ end
 --- Move a client.
 -- @param c The client to move, or the focused one if nil.
 -- @param snap The pixel to snap clients.
-function mouse.client.move(c, snap)
+-- @param finished An optional callback function, that will be called
+--                 when moving the client has been finished. The client
+--                 that has been moved will be passed to that function.
+function mouse.client.move(c, snap, finished)
```

Comment:
`_cb` is used by naughty and gears.cache already, so I would go with `finished_cb`.

While at it, then please also squash your commits into one, and then I think it's good to be merged.

Thanks!